ops/rackspace-agent-plugins/bf-rollups-delay.py

```diff
@@ -113,27 +120,34 @@ def print_stats_for_metrics_state(metrics_state_for_shards):
 
         if (len(across_shards_most_delay)):
             output[resolution] = max(across_shards_most_delay)
+        else:
+            output[resolution] = 0
 
     for resol, delay in output.items():
-        print 'metric %s float %f slots' % ('_'.join([resol, 'delay']), delay)
+        print 'metric %s uint32 %u' % ('_'.join([resol, 'delay']), delay)
+
 
+def main():
+    parser = argparse.ArgumentParser(description='For each rollup level, lists the number of slots which need to '
+                                                 'be processed by blueflood.  One day is approximately 300 slots.')
+    parser.add_argument( '-s', '--servers', help='Cassandra server IP addresses, space separated', required=True, nargs="+")
```

Comment:
Are these servers of the same cluster? If yes, what would be the benefit of passing in multiple servers? I am not critiquing, just trying to understand usage. :-)